lib/models/config.dart

```diff
@@ -182,6 +186,23 @@ class Config {
     }
     final bool admin = inno['admin'] ?? true;
 
+    if (inno['license_file'] != null && inno['license_file'] is! String) {
+      CliLogger.exitError("inno_bundle.installer_icon attribute is invalid "
+          "in pubspec.yaml.");
+    }
+
+    final licenseFile = File(inno['license_file'] != null
+                ? p.join(
+                    Directory.current.path, p.fromUri(inno['license_file']))
+                : p.join(Directory.current.path, 'LICENSE'))
+            .existsSync()
```

Comment:
Thanks for your contribution, I will look it up and refactor this part myself and merge it, much appreciated